### ISSUE TITLE
Add compatibility with PlayerEx

### ISSUE THREAD

**Issue by zndrmn** [Stereowalker/Survive#63]
[PlayerEx](https://www.curseforge.com/minecraft/mc-mods/playerex) adds a gui which aims to be an rpg styled gui.

The thirst and energy are not only not there, the whole gui disappears with both mods installed.

You should only have to add new gui elements to fit the theme.

[PlayerEx](https://www.curseforge.com/minecraft/mc-mods/playerex)

**Comment by Stereowalker** [comment]
It's only not showing because they prevented the vanilla food notches from rendering. I don't use PlayerEx, so it would be difficult for me to add gui elements based on its theme

**Comment by ConqAra** [comment]
you can disable playerex's gui in the config. I think adding support for this gui shouldn't be stereo's problem, but rather the developer for PlayerEx, detecting when the mod is installed and thirst is enabled. Should go to him about it.

**Comment by zndrmn** [comment]
Yeah I'm using it disabled. I've contacted him about the problem a while ago and came here. It's not the biggest deal to me, so I'm fine if you don't add compatibility with the mod because I can just disable the modded gui. I will tell him about what you said ConqAra some time this week. He did mention that it would be a bit tedious to support because it's not open source.

**Comment by marc-cwm** [comment]
I second the request for PlayerEx HUD compatibility in some fashion.

I really like the HUD, but I personally don't care if Survive matches the theme/GUI.  As an alternate option, I would be interested in a way to maybe decouple the thirst/etc elements from the vanilla food display/hotbar in general.  This also might fix a lot of other potential problems with various things that can sometimes overlap in the same position above the hunger bar.
